### ISSUE TITLE
Unable to deploy service 201-aci-wordpress with storage account type as Standard_ZRS.

### ISSUE THREAD

**Issue by Shahaji2016** [Azure/azure-quickstart-templates#7987]
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Sorry to hear you had a bad experience with one of the templates :worried: But, in case you're just asking a question, we're happy to help. You can also check if the question might already have been asked here https://github.com/Azure/azure-quickstart-templates/issues?utf8=%E2%9C%93&q=is%3Aissue

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

[Template Name goes here](Template link goes here)

### Issue Details

Unable to deploy service 201-aci-wordpress with storage account type as Standard_ZRS.

### Repro steps
1. Download template 201-aci-wordpress from https://github.com/Azure/azure-quickstart-templates/tree/master/201-aci-wordpress
2. Change defaultValue for storageAccountType to Standard_ZRS
3. Deploy service using az CLI command

**Comment by MCKLMT** [comment]
I don't encounter the issue using the deploy button in West Europe. What are the parameters used?

**Comment by Shahaji2016** [comment]
With Deploy button, you need to select Storage account Type as Standard_ZRS

**Comment by MCKLMT** [comment]
Yes, I used this parameter and West Europe for the location and the template is successfully deployed. Can you check again?

**Comment by MCKLMT** [comment]
Also please note that using ZRS storage requires Availability Zones which are not yet available in all regions. https://docs.microsoft.com/en-us/azure/availability-zones/az-region

**Comment by Shahaji2016** [comment]
Okay let me check once with West Europe location, it is failing for East US location, with the following error.

<img width="1217" alt="Screenshot 2020-08-19 at 10 14 03 AM" src="https://user-images.githubusercontent.com/17567976/90593255-20217980-e205-11ea-875b-2d8997e400eb.png">

**Comment by Shahaji2016** [comment]
@MCKLMT It is failing on West Europe too, I have the fix for the same but not able to Push my changes on this repository with new branch.

**Comment by MCKLMT** [comment]
I have reproduce the issue with the CI https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=31448&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=7bf03a3b-69d5-5660-244f-21e634d6903f

**Comment by Shahaji2016** [comment]
okay, thank you @MCKLMT, can I fix it or you are going to fix it?

**Comment by MCKLMT** [comment]
How would you fix it? The issue is because the container groups creating the file share. I suggest replacing these parts with deployment scripts which will create the file shares. Your thoughts?

**Comment by Shahaji2016** [comment]
For ZRS storage account type, supported kind value in **StorageV2** and for other, it is **Storage** so by adding if the condition for the kind parameter we can fix it.

**Comment by MCKLMT** [comment]
It's not enough. I changed the storagekind to V2 in my pull request and the deployment failed in US Gov.

**Comment by MCKLMT** [comment]
Can you test again with this template? https://raw.githubusercontent.com/MCKLMT/azure-quickstart-templates/201-aci-wordpress/201-aci-wordpress/azuredeploy.json
I have replaced the container group creating the file shares with two deployment scripts. I integrated these changes on my pull request and the CI is successful.

**Comment by Shahaji2016** [comment]
Okay sure let me try, also I have one doubt regarding type Microsoft.Authorization/roleDefinitions which is newly added in your PR, is this really required?

**Comment by MCKLMT** [comment]
Yes. The user identity is used to run the deployment scripts to create the two file shares.

**Comment by MCKLMT** [comment]
Can you close the issue?
